README.md

```diff
@@ -118,3 +122,115 @@ docs/CONTRIBUTING.md
 
 which contains important information.
 
+
+
+# PT-BR
+## Introdução
+Snaskii é um jogo de cobra ASCII muito simples, uma homenagem ao clássico
```

Comment:
I think that "jogo da cobrinha" is a more common name in Brazil, but "jogo de cobrea" is ok too.